core/src/main/java/fr/sncf/osrd/api/TimetableCacheManager.kt

```diff
@@ -127,15 +140,17 @@ class TimetableCacheManager(
     }
 
     @WithSpan(value = "Fetching timetable content", kind = SpanKind.SERVER)
-    private fun fetchTimetableRequirements(
+    private suspend fun fetchTimetableRequirements(
         infraId: String,
         infra: RawInfra,
         timetableId: TimetableId,
     ): STDCMRequirements {
-        val cacheFile = if (disableAllCaching) null else "$timetableId.cbor"
+        val cacheKey = "$timetableId-$infraId-$osrdGitDescribe"
```

Comment:
Nit: For consitency with editoast can we do:


```suggestion
        val cacheKey = "$osrdGitDescribe-$infraID-$timetableId"
```